google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -1101,10 +1101,9 @@ def ingest_from_gcs(
                     higher than millisecond precision.
             gcs_source_uris (Union[str, List[str]]):
                 Required. Google Cloud Storage URI(-s) to the
-                input file(s). May contain wildcards. For more
-                information on wildcards, see
-                https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
-                Example:
+                input file(s).
```

Comment:
None of the Dataset support wildcards. @nayaknishant are we sure that FeatureStore supports wildcards?

Reply:
hey Ivan, I'll take a look. I don't think so though.

Reply:
Thanks!

Reply:
Tested on https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/feature_store/sdk-feature-store.ipynb and seems to be working. We can keep the wildcards description for now.